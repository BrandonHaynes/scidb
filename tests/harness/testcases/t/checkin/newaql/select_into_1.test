--setup
--start-query-logging
create array A <a:int32> [x=0:2,3,0]
create array B <a:int32> [x=0:2,3,0]
create array C <a:int32> [x=0:2,1,0]
create array D <a:int32> [x=0:2,3,1]
create array E <a:int32> [x=0:2,2,1]

store(apply(build(<a:int32>[          x=0:2,3,0],x),b,true),Y)
store(build(<b:bool> [y=0:2,3,0,x=0:2,3,0],true),Z)

--test
--aql select * into A from build(A,x+10)
--aql select * into B from A
--aql select * into B from A where x<>1
--aql select * into C from A
--aql select * into C from A where x>1
--aql select * into D from A
--aql select * into D from A where x<1
--aql select * into E from A
--aql select * into E from A where x=1

--aql select * into Z from Y

--cleanup
remove(A)
remove(B)
remove(C)
remove(D)
remove(E)
remove(Y)
remove(Z)
--stop-query-logging
