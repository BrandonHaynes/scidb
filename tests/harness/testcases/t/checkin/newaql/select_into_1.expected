SCIDB QUERY : <create array A <a:int32> [x=0:2,3,0]>
Query was executed successfully

SCIDB QUERY : <create array B <a:int32> [x=0:2,3,0]>
Query was executed successfully

SCIDB QUERY : <create array C <a:int32> [x=0:2,1,0]>
Query was executed successfully

SCIDB QUERY : <create array D <a:int32> [x=0:2,3,1]>
Query was executed successfully

SCIDB QUERY : <create array E <a:int32> [x=0:2,2,1]>
Query was executed successfully

SCIDB QUERY : <store(apply(build(<a:int32>[          x=0:2,3,0],x),b,true),Y)>
{x} a,b
{0} 0,true
{1} 1,true
{2} 2,true

SCIDB QUERY : <store(build(<b:bool> [y=0:2,3,0,x=0:2,3,0],true),Z)>
{y,x} b
{0,0} true
{0,1} true
{0,2} true
{1,0} true
{1,1} true
{1,2} true
{2,0} true
{2,1} true
{2,2} true

SCIDB QUERY : <select * into A from build(A,x+10)>
{x} a
{0} 10
{1} 11
{2} 12

SCIDB QUERY : <select * into B from A>
{x} a
{0} 10
{1} 11
{2} 12

SCIDB QUERY : <select * into B from A where x<>1>
{x} a
{0} 10
{2} 12

SCIDB QUERY : <select * into C from A>
{x} a
{0} 10
{1} 11
{2} 12

SCIDB QUERY : <select * into C from A where x>1>
{x} a
{2} 12

SCIDB QUERY : <select * into D from A>
{x} a
{0} 10
{1} 11
{2} 12

SCIDB QUERY : <select * into D from A where x<1>
{x} a
{0} 10

SCIDB QUERY : <select * into E from A>
{x} a
{0} 10
{1} 11
{2} 12

SCIDB QUERY : <select * into E from A where x=1>
{x} a
{1} 11

SCIDB QUERY : <select * into Z from Y>
{y,x} b
{0,0} true
{0,1} true
{0,2} true

SCIDB QUERY : <remove(A)>
Query was executed successfully

SCIDB QUERY : <remove(B)>
Query was executed successfully

SCIDB QUERY : <remove(C)>
Query was executed successfully

SCIDB QUERY : <remove(D)>
Query was executed successfully

SCIDB QUERY : <remove(E)>
Query was executed successfully

SCIDB QUERY : <remove(Y)>
Query was executed successfully

SCIDB QUERY : <remove(Z)>
Query was executed successfully

